packages/jest-message-util/src/index.ts

```diff
@@ -10,6 +10,7 @@ import * as fs from 'graceful-fs';
 import type {Config, TestResult} from '@jest/types';
 import chalk = require('chalk');
 import micromatch = require('micromatch');
+import prettyFormat = require('pretty-format');
```

Comment:
add a dependency on this module - the linter doesn't understand the `import t = require()` syntax which is why it didn't warn you.

Also add it to `tsconfig.json`

Reply:
you also need to run `yarn` from root

Reply:
should be good now.